OWNERS

```diff
@@ -34,6 +34,7 @@
     {
       pattern: '{package.json,package-lock.json}',
       owners: [
+        {name: 'renovate-approve'},
```

Comment:
Instead of touching all these files, consider adding a new rule for `**/package{,-lock}.json` to apply recursively

Reply:
Good idea! Done.